src/webui/www/private/views/preferences.html

```diff
@@ -325,6 +325,10 @@
                 </td>
             </tr>
         </table>
+        <div class="formRow">
+            <input type="checkbox" id="proxy_for_hostnames_checkbox" />
```

Comment:
Add the tooltip here too by using the `title` attribute.

Reply:
Done.